v2/sourcedb-to-spanner/terraform/samples/sharded-bulk-migration/README.md

```diff
@@ -213,6 +213,37 @@ There can be multiple reasons for this.
     - Consider bumping up `maxConnection` and `numPartition` parameters when launching the jobs. We recommend `320` for
       maxConnections and `10000` for numPartitions.
 
+### Allowing network connectivity
+
+Ensure Dataflow VMs are able to access the MySQL instance. This would require:
+
+- Configuring MySQL to allow connections from the Dataflow IPs.
+- Configuring the firewall to allow ingress TCP connection on the MySQL port
+  from the Dataflow IPs.
+
+#### Configuring connectivity on Cloud SQL
+
+Cloud SQL allows multiple ways to establish connectivity. With google
+private access enabled, Dataflow should automatically be able to [access
+the instance](https://cloud.google.com/sql/docs/mysql/authorize-networks).
```

Comment:
This is not true IMO. Have you validated this?

Reply:
THis should be within the same VPC. Rephrased.